doc/api/next_api_changes/2019-02-07-AL.rst

```diff
@@ -0,0 +1,4 @@
+Deprecations
+````````````
+
+``axis3d.get_flip_min_max`` is deprecated.
```

Comment:
Oh wait, that's just code style; why _not_ the full path then?

Reply:
Added mplot3d in front; as a rule I tend to not put the leading "matplotlib" or "mpl_toolkits" (it's a bit repetitive otherwise).
It's code style because the link is about to be broken.